doc/guide/guide-bloch.rst

```diff
@@ -407,7 +407,7 @@ An example of generating images for generating an animation outside of Python is
 
 Generating an animation using ffmpeg (for example) is fairly simple::
 
-   ffmpeg -r 20 -b 1800 -i bloch_%01d.png bloch.mp4
+   !ffmpeg -r 20 -i temp/bloch_%01d.png -b:v 1800 bloch.mp4
```

Comment:
The `!` is necessary inside a Jupyter notebook, but not elsewhere, so I think we should drop it.

The new command runs for me (great!) but the video is just a constant grey square, so I think there are still other problems to be solved. Does the video work for you?

Reply:
Thank you so much for the review @hodgestar ! I have made the required changes by removing `!` and adding a constant rate factor so that the video renders.

I have checked the file locally, and now the video does seem to render! Thanks for pointing this out.

Reply:
I know it's gone now, but for the future: having the `!` prefix in bash actually means "recall the last time this command was used", so you copied it directly, you'd actually end up running `ffmpeg` with two sets of arguments!  For example:
```bash
$ echo hello
hello
$ !echo world
hello world
```

In terminal text editors (vim, ipython, etc), the `!` prefix common for "run this command in a subshell and insert the output".

Reply:
Thank you for correcting me regarding `!` and its unintended consequences.